app/src/main/res/xml/status_widget_info.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<appwidget-provider xmlns:android="http://schemas.android.com/apk/res/android"
+    android:description="@string/widget_description"
+    android:initialKeyguardLayout="@layout/status_widget"
+    android:initialLayout="@layout/status_widget"
+    android:minWidth="@dimen/widget_min_size"
+    android:minHeight="@dimen/widget_min_size"
```

Comment:
Are these actually required?

Reply:
The widget works same way without them, so they look redundant.
However I couldn't find official defaults for `minWidth`/`minHeigth` or `targetCellWidth`/`targetCellHeight` (these are for Android 12+).

Reply:
These and related constants will be removed.